lib/bolt/application.rb

```diff
@@ -413,6 +413,168 @@ def list_plugins
       { plugins: plugins.list_plugins, modulepath: pal.user_modulepath }
     end
 
+    # Applies one or more policies to the specified targets.
+    #
+    # @param policies [String] A comma-separated list of policies to apply.
+    # @param targets [Array[String]] The list of targets to apply the policies to.
+    # @param noop [Boolean] Whether to apply the policies in no-operation mode.
+    # @return [Bolt::ResultSet]
+    #
+    def apply_policies(policies, targets, noop: false)
+      policies = policies.split(',')
+
+      # Validate that the policies are available to the project.
+      unavailable_policies = policies.reject do |policy|
+        @config.policies.any? do |known_policy|
+          File.fnmatch?(known_policy, policy, File::FNM_EXTGLOB)
+        end
+      end
+
+      if unavailable_policies.any?
+        command = Bolt::Util.powershell? ? 'Get-BoltPolicy' : 'bolt policy show'
+
+        # CODEREVIEW: Phrasing
+        raise Bolt::Error.new(
+          "The following policies are not available to the project: '#{unavailable_policies.join("', '")}'. "\
+          "Policies must be listed in a project's 'policies' setting before they are applied to targets. "\
```

Comment:
```suggestion
          "You must list policies in a project's 'policies' setting before Bolt can apply them to targets. "\
```
Just taking out some passive voice